clients/web/src/collections/Collection.js

```diff
@@ -70,6 +107,11 @@ var Collection = Backbone.Collection.extend({
      * result when displaying it to the user.
      */
     pageNum: function () {
+        if (this.filterFunc) {
+            if (this.append) { return 0; }
+            this.pageOffsetStack = this.pageOffsetStack || [];
+            return this.pageOffsetStack.length - 1;
```

Comment:
I can change this if you want, but I think it does make some sense.  Similar to in the two-pops question, you only run into this -1 result if you call pageNum before ever fetching any pages.  It's only after having fetched the first page that you have the information you need to display pageNum() === 0.  One could interpret pageNum() === -1 as signaling that you're not even *on* a page which is distinct from being on the pageNum() === 0 page.

Anyway, let me know if you still want me to change this.